lib/Logger/Logger.cpp

```diff
@@ -45,19 +47,16 @@ uint16_t Logger::getPort() {
     return Logger::getInstance().getPort();
 }
 
-void Logger::log(const Level level,
-                    const String& file,
-                    const __FlashStringHelper* function,
-                    uint32_t line,
-                    const char* logmsg) {
+void Logger::log(const Level level, const String& file, const __FlashStringHelper* function, uint32_t line,
+                 const char* logmsg) {
```

Comment:
This could be one line or should be split by "," like before.
i do prefer the split, like with our "editableVars".
if we go with one line, we have to find a way with the very long mashine state stuff.

Reply:
Split is nicer, I agree. Added via

```
AlignAfterOpenBracket: Align
BinPackParameters: false
```